internal/vds/vds.cpp

```diff
@@ -392,30 +405,17 @@ struct response fetch_horizon(
     std::size_t i = 0;
```

Comment:
> replacementNoValue – If specified, this value is used to replace regions of the input VDS that has no data.

This only applies to areas within the volume that OpenVDS has explicitly marked as missing (with the tracemask I think). It will then use this replacementNoValue for these dead traces. And then it will interpolate normally on top of that I believe. For samples out of bounds it will interpolate infinite I think

(I'm not 100% sure about this, but it definitely doesn't do what we need)